docs/learn/smart-contract-developers/create-nft-collection-with-lsp7-tokenId.md

```diff
@@ -0,0 +1,953 @@
+
+## Create the LSP7 token
+
+Even though it seems a little bit counter intuitive, we will start by creating the LSP7 token. The reason is that the LSP8 collection will be in charge of deploying the LSP7 sub-collection tokens.
+
+### Specify what we need to be set at deployment
+
+To create the LSP7 token contract, we will need to know what we want to be set at deployment. In our case, we want to set the following:
+
+- the name of the sub-collection
+- the symbol of the sub-collection
+- the address of the owner of the sub-collection (in our case it will the the LSP8 collection contract)
+- the initial receiver of the tokens that will be minted at deployment (could be the token creator)
+- the number of tokens that can be minted
+- whether the token is divisible or not
+- the total supply of the token
+- the [Token Type](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4tokentype)
+- the [LSP4 Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4metadata)
+
+This is just an example but you may want to set a deployment of other parameters such as the [LSP4 Creators array](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md#lsp4creators).
+
+### Imports
+
+Let's start creating an `LSP7Token.sol` file in the `contracts` folder and importing:
+
+- the [`LSP7DigitalAsset.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol) contract
+- the [`LSP4DigitalAssetMetadata.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP4DigitalAssetMetadata/LSP4DigitalAssetMetadata.sol) contract
+- the [`_LSP4_SUPPORTED_STANDARDS_KEY`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#supportedstandardslsp4digitalasset) constant
+- the [`_LSP4_SUPPORTED_STANDARDS_VALUE`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#supportedstandardslsp4digitalasset) constant
+- the [`_LSP4_METADATA_KEY`](../../standards//tokens/LSP4-Digital-Asset-Metadata.md#lsp4metadata) constant
+- the [`_LSP8_REFERENCE_CONTRACT`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8referencecontract) constant
+
+```typescript
+import { LSP7DigitalAsset } from '@lukso/lsp-smart-contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.sol';
+import { LSP4DigitalAssetMetadata } from '@lukso/lsp-smart-contracts/contracts/LSP4DigitalAssetMetadata/LSP4DigitalAssetMetadata.sol';
+import {
+  _LSP4_SUPPORTED_STANDARDS_KEY,
+  _LSP4_SUPPORTED_STANDARDS_VALUE,
+  _LSP4_METADATA_KEY,
+} from '@lukso/lsp-smart-contracts/contracts/LSP4DigitalAssetMetadata/LSP4Constants.sol';
+import { _LSP8_REFERENCE_CONTRACT } from '@lukso/lsp-smart-contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol';
+```
+
+### Contract
+
+At deployment, we will need to set the following parameters:
+
+- `name`: the name of the token
```

Comment:
why `name` and not `name_` as per code snippet below?

I'd keep it consistent and use `name_`, `symbol_`, etc. here so it matches with the code